src/toniarts/openkeeper/world/creature/steering/AbstractCreatureSteeringControl.java

```diff
@@ -47,8 +47,8 @@
     private boolean tagged;
     private boolean independentFacing = false;
     private float maxLinearSpeed = 1;
-    private float maxLinearAcceleration = 2;
-    private float maxAngularSpeed = 10.0f;
+    private float maxLinearAcceleration = 4;
+    private float maxAngularSpeed = 20.0f;

```

Comment:
How is the steering now?

Reply:
When this code worked, it was better. Now it needs to be transferred to another class. I don’t know which one. I wrote down here so as not to forget

Reply:
Ah, oh yeah, true. I thought this was the class we used. Yep, this is deprecated as well. The new one is in toniarts.openkeeper.game.navigation.steering.*